src/main/webapp/add_event_page/script.js

```diff
@@ -15,8 +44,32 @@ function initMap(){
         var latLng = marker.latLng; 
         currentLatitude = latLng.lat();
         currentLongitude = latLng.lng();
-        jQ("#latitude").val(currentLatitude);
-        jQ("#longitude").val(currentLongitude);
     }); 
 }
-window.initMap = initMap;
\ No newline at end of file
+window.initMap = initMap;
+
+function submitEvent(){
+    var event_name = document.getElementById("name").value;
+    var event_des = document.getElementById("description").value;
+    var event_loc = document.getElementById("location_name").value;
+    var event_date = new Date(document.getElementById("date").value).toISOString();
+    var event_type = document.getElementById("event_type").value;
```

Comment:
We can check if the event type is `Social` and if so make the subject an empty string.